docs/components/movement-sensor/merged.md

```diff
@@ -0,0 +1,107 @@
+        "linear_acceleration": ["<your-accelerometer-sensor-name-1>"]
+      },
+    "depends_on": [] 
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+// { "components": [ { ... },
+{
+    "name": "<your-sensor-name>",
+    "type": "movement_sensor",
+    "model": "merged",
+    "attributes": {
+        "position": ["gps1"],
+        "orientation" : ["vectornav"],
+        "compass_heading" : ["gps1"],
+        "angular_velocity":["vectornav", "mpu6050"],
+        "linear_velocity": ["gps1"],
+        "linear_acceleration": ["adxl345"]
+    }, 
+      "depends_on":[]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+Configure an array of the `name` of each movement sensor you want to add to your robot as a merged resource in the attributes of the `merged` movement sensor model:
+
+- The name of each attribute represents the `Property` that that particular movement sensor supports, or the type of reading or measurement that it takes.
+- Get the properties supported by each model from its [model configuration documentation](/components/movement-sensor/#configuration), or by calling [`GetProperties()`](/components/movement-sensor/#getproperties) on the sensor.
+- Use this information to determine which attribute to put its `name` inside the array of.
+You can use the same sensor for multiple attributes if it supports multiple properties.
+
+Name | Type | Inclusion | Description
+---- | ---- | --------- | -----------
+`position` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads position. |
+`orientation` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads orientation. |
+`compass_heading` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads compass heading position. |
+`linear_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear velocity. |
+`angular_velocity` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads angular velocity. |
+`linear_acceleration` | array | **Dependent on Readings Type Supported** | The `name` of the movement sensor you want to merge, if it reads linear acceleration |
+
+Note that only one sensor from each array can be used to retrieve each type of reading.
+The first sensor in the array that has implemented the relevant API method in its model and does not raise an error at runtime is used by your robot to get that type of reading.
+For instance, in the **JSON Example** above, if both `"vectornav"` and `"mpu6050"` support returning `angular_velocity`, `"mpu6050"` is only used to read linear acceleration on the robot if `"vectornav"` returns an error at runtime.
```

Comment:
```suggestion
For instance, in the **JSON Example** above, if both `"vectornav"` and `"mpu6050"` support returning `angular_velocity`, `"mpu6050"` is only used to read angular velocity on the robot if `"vectornav"` returns an error at runtime.
```

Reply:
done... 
Idk why my brain wants to switcheroo, thanks greatly for catching